feature/community/src/main/java/com/iteneum/community/domain/CardMenuItem.kt

```diff
@@ -0,0 +1,9 @@
+package com.iteneum.community.domain
+
+enum class CardMenuItem {
```

Comment:
- Naming convention: The enum values are in all caps, which is the recommended naming convention for constants in Kotlin. However, the naming could be improved by separating the words with underscores, like "COMMUNITY_WALL" instead of "COMMUNITYWALL".

- Ordering: The enum values are listed in no particular order. It's a good practice to order them alphabetically to make them easier to read and maintain.

- Consider adding a description or comment for the enum class to clarify its purpose:

- Consider renaming the CardMenuItem class to a more descriptive name that reflects the fact that it represents a list of community sections. For example, CommunitySectionList

Reply:
will not change this, cos of sealed class that will be used

Reply:
Being something that is kept in the project, we must try to keep the code clean, always following the best practices, update the observations made in that class.

Reply:
Sorry I was not clear enough  i am not using enum class anymore due to sealed class, the other suggestions will read it carefully and take care of the things that have to change,  please consider open different comments for different changes  this one is cluttered of information  did not read the other bullets

Reply:
Update your branch with the new changes, otherwise for me it's as if you haven't made any changes yet

Reply:
Ordering: It is ordered as it is needed to be presented in the presentation, even it is not necessary and filtered through a when  component, but will be ordered in ascendent  from A to Z

adding  description

Renamed to  CommunitySectionList